### PR TITLE
Feature/nav toggle bug fix

### PR DESCRIPTION
**Pull request by katieolson84**
## Description

Quick bug fixes for navigation sidebar. Moved the button styling to the less file. Updated the color of the arrows and darkened the gray a little. Moved the 'Review Cases' to the admin drop-down. 

## Type of change

Please delete options that are not relevant.

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X]  No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/SideDrawer/SideDrawer.js [hunk 1]
```diff
@@ -34,16 +34,7 @@ function SideDrawer() {
...
-          onClick={toggleCollapsed}
-          type="primary"
-          style={{
-            marginBottom: 16,
-            backgroundColor: '#f4f6f7',
-            marginLeft: 16,
-            color: '#2a5c8d',
-          }}
-        >
+        <Button type="primary" onClick={toggleCollapsed}>
```
Comment (paulstgermain) [inline src/components/pages/SideDrawer/SideDrawer.js:37]: Good work moving the styling to the LESS file, keeping the separation of concerns consistent and the component code short.

File: src/components/pages/SideDrawer/_SideDrawer.less [hunk 2]
```diff
@@ -14,31 +14,32 @@
 @home-blue: '../../../styles/icons/home-blue.svg';
 
 @selected-color: #2a5c8d;
-@gray: #cacccf;
+@gray: #939292;
+@light-gray: #cacccf;
 @light-blue: #a0c5dd;
+@white: #ffffff;
```
Comment (paulstgermain) [inline src/components/pages/SideDrawer/_SideDrawer.less:20]: LOVE seeing the use of LESS variables, they make life so much easier!

**Review by paulstgermain — APPROVED** [review]
Short, sensible, clean addition to the codebase, good work!

**Review by ChadWCox — APPROVED** [review]
looking good katie. thx

**Pull request merged**